drt: cleanup workload runner scripts

ajstorm opened the pull request:
Cleanup the workload runner scripts to:

- Add the chaos test to systemd-run
- Fix a few errors in cct_tpcc_drop
- Add kv and tpcc scripts

Release note: none
Epic: none

ajstorm committed:
drt: cleanup workload runner scripts

Cleanup the workload runner scripts to:

- Add the chaos test to systemd-run
- Fix a few errors in cct_tpcc_drop
- Add uid/gid to start_all_workloads

Release note: none
Epic: none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/120493)
<!-- Reviewable:end -->

ajstorm committed:
drt: add additional scripts

Adding scripts for kv, schemachange and tpcc workloads.

Release note: none
Epic: none

itsbilal commented on pkg/cmd/drt/scripts/start_all_workloads.sh:
```diff
@@ -2,3 +2,4 @@ sudo systemd-run --working-directory=/home/ubuntu --service-type exec --collect
 sudo systemd-run --working-directory=/home/ubuntu --service-type exec --collect --unit cct_tpcc ./tpcc_run.sh
```
Let's add uids/gids for all of these, as they all need it now

ajstorm replied:
Done.  I also added the schemachange workload.  PTAL.

itsbilal reviewed (approved):
LGTM

ajstorm commented:
bors r=itsbilal
